src/lib/es2015.iterable.d.ts

```diff
@@ -135,7 +135,7 @@ interface MapConstructor {
 interface WeakMap<K extends object, V> { }
 
 interface WeakMapConstructor {
-    new <K extends object = object, V = any>(iterable: Iterable<[K, V]>): WeakMap<K, V>;
+    new <K extends object, V>(iterable: Iterable<[K, V]>): WeakMap<K, V>;

```

Comment:
Because this PR only changed this `WeakMap` constructor overload, <https://github.com/microsoft/TypeScript/issues/23551> has resurfaced once again, and the following is still valid:
```ts
new WeakMap<object>();
```

---

I’m fixing that in <https://github.com/microsoft/TypeScript/pull/41635>.